ethereum/eth/src/main/java/org/hyperledger/besu/ethereum/eth/sync/snapsync/SnapSyncDownloader.java

```diff
@@ -48,7 +48,7 @@ public SnapSyncDownloader(
 
   @Override
   protected CompletableFuture<FastSyncState> start(final FastSyncState fastSyncState) {
-    LOG.debug("Start snap sync with initial sync state {}", fastSyncState);
+    LOG.info("Start snap sync with initial sync state {}", fastSyncState);
```

Comment:
the same here we need to be sure this log is not spamming on mainnet sync

Reply:
I've reverted it to a `debug` log